.github/workflows/documentation.yml

```diff
@@ -0,0 +1,57 @@
+# yamllint disable
+env:
+  PYTHON_PACKAGE_MANAGER: "conda" # Python package manager to create the python environments
+
+name: documentation
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
```

Comment:
I suppose this is empty on purpose?

Reply:
Yes, this is simply the way of listing various trigger signals when one of them has suboptions specified (like `push` does)